README.md

```diff
@@ -65,11 +65,11 @@ Next, you need to download a recent
 [labsjdk](http://www.oracle.com/technetwork/oracle-labs/program-languages/downloads/index.html).
 Extract it inside the `sulong-dev` directory:
 
-    tar -zxf labsjdk-8u92-jvmci-0.17-linux-amd64.tar.gz
+    tar -zxf labsjdk-8u92-jvmci-0.20-linux-amd64.tar.gz
```

Comment:
There is already an open PR for fixing the JVMCI version in the documentation: #425